projects/kit/src/lib/time/types/time-mode.ts

```diff
@@ -0,0 +1 @@
+export type MaskitoTimeMode = 'HH:MM' | 'HH:MM:SS' | 'HH:MM:SS.MSS';
```

Comment:
That's what I asked about in parsing function. I think we should also support `SS.MSS` and `MM.SS.MSS`. Maybe in the next iteration, but keep that in mind.

Reply:
Okey, let's keep it in mind for the next iteration.

For the first iteration, all `kit`-masks should reproduce already existing Taiga UI features + fixing old bugs.
All new features – next iteration.

Reply:
- https://github.com/Tinkoff/maskito/issues/38